Audio PR 3 - One Augmentation support [PreEmphasis Filter]

swetha097 opened the pull request:
Contains the changes needed in rocAL for PreEmphasis Filter support

Commit:
PreEmphasis Filter

Commit:
Add Preemp filter augmentations files

Commit:
Resolving PR comments

Commit:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

Commit:
Resolve PR comments

shobana-mcw reviewed (commented):
Address the review comments.

Commit:
Add basic testing for Audio to compare Bin files

Commit:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

swetha097 committed:
Update rocAL_api_audio_unittest.py

Commit:
Add comment

Commit:
Fix formatting issues - Minor changes

Commit:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

Commit:
Minor changes - Modifying the names of the arguments

Commit:
Merge remote-tracking branch 'origin/swbs/audio/pr2' into swbs/audio/pr3

Commit:
Added a bried desc for the rocAL enum for border type

swetha097 reviewed (commented):
> Please fix build errors and unit test error in this PR 

Fixed and merged the latest changes from prev base PR too which resolves the conflicts

Commit:
Add a WRN statement in PreEmphasis Filter to only use FP32 dtype

Commit:
Change the borderType enum to int32 from uint32 dtype

swetha097 committed:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

Commit:
Remove the reset_tensor_roi() from the PreEmphasis augmentation making sure the src and dst roi points to the same location

Commit:
Del the Unit Test Files introduced earlier

shobana-mcw reviewed (commented):
Please address the review comments. 
I am not seeing the python unit test for pre emphasis filter. 
Please add that too in this PR.

swetha097 committed:
Add pre_emphasis function and gollden output comparison in audio unit tests

swetha097 committed:
minor change - add update val in create array

swetha097 committed:
minor change

fiona-gladwin committed:
Update unit test

fiona-gladwin committed:
Minor change

swetha097 committed:
minor change

swetha097 committed:
Merge branch 'swbs/audio/pr3' of https://github.com/swetha097/rocAL into swbs/audio/pr3

fiona-gladwin committed:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

fiona-gladwin committed:
Add file reader to python audio unit test

fiona-gladwin committed:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

Commit:
Remove the unused variable output - Resolve warnings in cpp unit test

Commit:
Remove the dst_roi arg passed to rpp

fiona-gladwin committed:
Minor changes

Change borderType enum prefix

fiona-gladwin committed:
Merge branch 'swbs/audio/pr3' of https://github.com/swetha097/rocAL into swbs/audio/pr3

SundarRajan28 committed:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/audio/pr2' into swbs/audio/pr3

swetha097 committed:
Update the changelog.md

swetha097 committed:
Merge remote-tracking branch 'swe_fork/swbs/audio/pr2' into swbs/audio/pr3

swetha097 committed:
Fix merge from PR 2

SundarRajan28 committed:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

shobana-mcw reviewed (commented):
Address the review comments.

fiona-gladwin committed:
Merge branch 'swbs/audio/pr2' into swbs/audio/pr3

swetha097 committed:
Resolve the PR comments

swetha097 committed:
Merge branch 'swbs/audio/pr3' of https://github.com/swetha097/rocAL into swbs/audio/pr3

kiritigowda committed:
Update rocAL-setup.py (#138)

fiona-gladwin committed:
Audio PR - rocAL Python changes for Audio Decoder (#120)

* Audio Decoder Python Changes

---------

Co-authored-by: root <root@ixt-sjc2-54.local.lan>
Co-authored-by: Swetha B S <swetha@multicorwareinc.com>
Co-authored-by: swetha097 <59434434+swetha097@users.noreply.github.com>
Co-authored-by: swetha097 <swetha@multicorewareinc.com>
Co-authored-by: Swetha B S <>
Co-authored-by: SundarRajan28 <sundarrajan@multicorewareinc.com>

fiona-gladwin committed:
Merge branch 'swbs/audio/pr2' of https://github.com/swetha097/rocAL into swbs/audio/pr3

fiona-gladwin committed:
Merge branch 'swbs/audio/pr3' of https://github.com/swetha097/rocAL into swbs/audio/pr3

fiona-gladwin committed:
Merge branch 'develop' of https://github.com/ROCm/rocAL into swbs/audio/pr3

LakshmiKumar23 committed:
cupy fall back cpu (#137)

swetha097 reviewed (commented):
> Why it's been set to None? We can have audio decoder as the default. 

Only then it runs all test cases by deafult

swetha097 committed:
Minor changes

swetha097 committed:
Address the PR comments

swetha097 committed:
Address Review comments

dgaliffiAMD committed:
Update CODEOWNERS

Fixed typo in the global owners definition.

peterjunpark committed:
Docs: fix broken links (#141)

* Fix broken links and sphinx errors

* Make 'How to' heading consistent

peterjunpark committed:
Add 6.1.0 link fix to develop (#143)

* Fix link to types.py

* Fix imagenet training link

kiritigowda committed:
RocAL-setup.py - traceback

Find Failure

LakshmiKumar23 committed:
Setup - add wheel install (#145)

kiritigowda committed:
Setup - Updates (#146)

SundarRajan28 committed:
Fixing cupy import issues (#148)

SundarRajan28 committed:
Unit Test changes for new rocal data (#150)

* Adding unit test changes for new rocal data

* Adding changes for python unit tests

* Updating python unit tests with correct dims

fiona-gladwin committed:
Introduce flag in CMake to enable Audio support (#149)

* Introduce CMake flag to enable Audio support

* Update README

SundarRajan98 committed:
Adding last batch policy for image loaders (#140)

* Adding last batch policy to rocAL

* Adding fixed for last batch policy

* Reverting changes

* Removing commented code

* Resolving review comments

* Minor changes

* Resolving review comments

* Minor changes

* Added comments

* Add comments in the file_source_reader.cpp

* Update file_source_reader.cpp

* Minor changes

* Update comment

* Resolving review comments

* Moving last batch policy to readers

---------

Co-authored-by: SundarRajan28 <sundarrajan@multicorewareinc.com>
Co-authored-by: sbavasab <102275787+sbavasab@users.noreply.github.com>

kiritigowda committed:
ROCm 6.2 updates (#152)

* ROCm - usecase rocm only

* BUILD_WITH_AMD_ADVANCE - Set by rocm version only

* Docker - updates

* Docker - updates

* Docker - Install rocDecode

* Docker - remove i386

* With rocDecode Deps no backward compati

* Changelog updates

LakshmiKumar23 committed:
Docker updates (#151)

* docker updates

* clean up

* upgrade pip on docker

* poytest version upgrade for dockers

* upgrade packge versions protobuf, pybind and turbojpeg

* merge conflicts

* merge conflicts - remove rpp source build

swetha097 committed:
Merge remote-tracking branch 'open_source/develop' into swbs/audio/pr3

kiritigowda committed:
Readme - Updates (#153)

* Readme - Updates

* Setup - Cleanup

* CMakeLists - remove OBSOLETES

* Readme - updates

* Updates

samjwu committed:
Docs - Update Read the Docs configuration to use Python 3.10 and latest rocm-docs-core (#158)

dependabot[bot] committed:
Docs - Bump rocm-docs-core[api_reference] from 1.2.0 to 1.2.1 in /docs/sphinx (#159)

Bumps [rocm-docs-core[api_reference]](https://github.com/RadeonOpenCompute/rocm-docs-core) from 1.2.0 to 1.2.1.
- [Release notes](https://github.com/RadeonOpenCompute/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/RadeonOpenCompute/rocm-docs-core/compare/v1.2.0...v1.2.1)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

LakshmiKumar23 committed:
Audio - Version check  (#156)

* version check from MIVisionX vx_rpp for Audio

* check if version file exists

* check for vx_rpp version and sndfile before settign rocal_audio

* remove unnecessary find_pacakge

* check for sndfile only if vx_rpp supports audio

dependabot[bot] committed:
Docs - Bump rocm-docs-core[api_reference] from 1.2.1 to 1.3.0 in /docs/sphinx (#163)

Bumps [rocm-docs-core[api_reference]](https://github.com/RadeonOpenCompute/rocm-docs-core) from 1.2.1 to 1.3.0.
- [Release notes](https://github.com/RadeonOpenCompute/rocm-docs-core/releases)
- [Changelog](https://github.com/ROCm/rocm-docs-core/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/RadeonOpenCompute/rocm-docs-core/compare/v1.2.1...v1.3.0)

---
updated-dependencies:
- dependency-name: rocm-docs-core[api_reference]
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

kiritigowda committed:
Merge branch 'develop' into swbs/audio/pr3

kiritigowda committed:
Setup - Updates (#164)

* Setup - Updates

* Setup - pip packages updates

* CMakeList - Add Protobuf deps

kiritigowda committed:
Merge branch 'develop' into swbs/audio/pr3
